src/sql/workbench/browser/designer/designer.ts

```diff
@@ -282,6 +283,9 @@ export class Designer extends Disposable implements IThemable {
 
 
 	public setInput(input: DesignerComponentInput): void {
+		if (this._input) {
+			this.submitPendingChanges();
```

Comment:
[sqllint        ]   287:4  error  Promises must be handled appropriately or explicitly marked as ignored with the `void` operator  @typescript-eslint/no-floating-promises